easybuild/easyconfigs/p/Platanus/Platanus-1.2.4-foss-2017a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+
+name = 'Platanus'
+version = '1.2.4' 
+
+homepage = 'http://platanus.bio.titech.ac.jp/'
+description = """PLATform for Assembling NUcleotide Sequences"""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['http://platanus.bio.titech.ac.jp/?ddownload=150']
+sources = ['%(name)s_v%(version)s.tar.gz']
+
+# fix Makefile to support newer gcc version
+patches = ['Platanus_new_gcc.patch']
```

Comment:
Ok, seems better. A question anyway, in the Makefile there are other flags like:

    -funroll-loops -Wall -finline-limit-50000

Are they managed by eb in any way?